messages2/src/protocols/mod.rs

```diff
@@ -0,0 +1,2 @@
+pub mod traits;
+pub mod nameless;
```

Comment:
`payload` is a decent name I think. It would be great if we could name both `crate::msg_types::types` and `crate::protocols::nameless` the same. What do you think about something like "data_impls"?

Reply:
Renamed modules so the directory structure now looks like:
- src
    - msg_types
        - protocols
    - msg_fields
        - protocols